config/jobs/kubernetes/sig-network/sig-network-misc.yaml

```diff
@@ -0,0 +1,255 @@
+presubmits:
+  kubernetes/dns:
+  - name: pull-kubernetes-dns-test
+    agent: kubernetes
+    context: pull-kubernetes-dns-test
+    branches:
+    - master
+    rerun_command: "/test pull-kubernetes-dns-test"
+    trigger: "(?m)^/test( all| pull-kubernetes-dns-test),?(\\s+|$)"
+    always_run: true
+    labels:
+      preset-service-account: "true"
+    spec:
+      containers:
+      - image: gcr.io/k8s-testimages/gcloud-in-go:v20171113-192bec25
+        args:
+        - --repo=k8s.io/$(REPO_NAME)=$(PULL_REFS)
+        - --root=/go/src
+        - --upload=gs://kubernetes-jenkins/pr-logs
+        - --scenario=execute
+        - --
+        - "true"
+
+periodics:
+- interval: 30m
+  agent: kubernetes
+  name: ci-kubernetes-e2e-gce-alpha-api
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+  spec:
+    containers:
+    - args:
+      - --timeout=80
+      - --bare
+      - --scenario=kubernetes_e2e
+      - --
+      - --check-leaked-resources
+      - --env-file=jobs/env/ci-kubernetes-e2e-gce-alpha-api.env
+      - --extract=ci/latest
+      - --gcp-master-image=gci
+      - --gcp-node-image=gci
+      - --gcp-nodes=4
+      - --gcp-project=k8s-e2e-gce-alpha-api-access
+      - --gcp-zone=us-central1-f
+      - --ginkgo-parallel=30
+      - --provider=gce
+      - --test_args=--ginkgo.focus=\[Feature:GCEAlphaFeature\] --minStartupPods=8
+      - --timeout=60m
```

Comment:
Is this timeout still necessary? I think I mistakenly removed all of them in https://github.com/kubernetes/test-infra/pull/8945 because I thought they are not needed any more :/